### PR TITLE
update vue quickstart instructions

### PR DESCRIPTION
**Pull request by alisaduncan**
## Description:
- **What's changed?** Update the Vue quickstart guide to use Vue 3, updated sdk usages, and better match code samples so devs have a better experience getting going.
- **Is this PR related to a Monolith release?** No, I don't think so

### Resolves:

* No Jira ticket yet

### CODE REVIEW

**Comment by chrisdmills-okta** [comment]
@shuowu-okta another one that we'd love your help on, if you have the time.

File: packages/@okta/vuepress-site/docs/guides/sign-into-spa/main/vue/reqauthspecific.md [hunk 1]
```diff
@@ -1,30 +1,21 @@
-Set the `requiresAuth` property to `true` in the optional `meta` object as part of the route definition.
+Set the `requiresAuth` property to `true` in the optional `meta` object as part of the route definition. Add `navigationGuard` function from `okta-vue` as a global guard to the router.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-spa/main/vue/reqauthspecific.md:1]:
```suggestion
Set the `requiresAuth` property to `true` in the optional `meta` object as part of the route definition. Add the `navigationGuard` function from `okta-vue` as a global guard to the router.
```

**Review by shuowu-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/guides/sign-into-spa/main/vue/reqauthspecific.md (hunk 1 above)
Comment (chrisdmills-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-spa/main/vue/reqauthspecific.md:1]: I think the wording could be clearer here. Would "Add the `navigationGuard` function from `okta-vue` to the router as a global guard." still make sense?

File: packages/@okta/vuepress-site/docs/guides/sign-into-spa/main/vue/config.md [hunk 2]
```diff
@@ -1,25 +1,26 @@
-First, create an [OktaAuth](https://github.com/okta/okta-auth-js) object with your configuration and pass it to the `okta-vue` plugin. You need to use `okta-vue` plugin after `vue-router`.
+First, create an [OktaAuth](https://github.com/okta/okta-auth-js) object with your configuration and pass it to the `okta-vue` plugin. You need to use `okta-vue` plugin after adding the router to the app.
```
Comment (chrisdmills-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-spa/main/vue/config.md:1]: "to use `okta-vue` plugin" -> "to use the `okta-vue` plugin"?

**Review by chrisdmills-okta — CHANGES_REQUESTED** [review]
@alisaduncan just a couple of small stylistic comments for you to address here.

**Comment by alisaduncan** [comment]
@chrisdmills-okta I addressed both comments and rebased to latest

**Review by chrisdmills-okta — APPROVED** [review]
Perfect, thanks @alisaduncan ! Let's put this one to bed.

**Pull request merged**